Makefile

```diff
@@ -8,10 +8,10 @@ YFLAGS = -q
 LEX = flex
 LFLAGS = --header-file=lexer.h
 
-SOURCES = parser.c lexer.c g-compiler.c
+SOURCES = parser.c lexer.c memory.c g-compiler.c
 OBJECTS = $(patsubst %.c, %.o, $(SOURCES))
 
-.PHONY: release lemon clean
```

Comment:
This makes the Makefile works as expected. If already compiled do not recompile.